pylint/checkers/base/basic_checker.py

```diff
@@ -444,6 +451,12 @@ def visit_expr(self, node: nodes.Expr) -> None:
             self.add_message("pointless-string-statement", node=node)
             return
 
+        # Warn W0133 for exceptions that are used as statements
+        if isinstance(expr, nodes.Call):
+            inferred = utils.safe_infer(expr)
+            if isinstance(inferred, objects.ExceptionInstance):
```

Comment:
One question I had is whether checking against the `objects.ExceptionInstance` type is safe, in terms of API stability - I didn't find it listed in the documentation (and could raise a documentation update as a separate issue and/or pull request, if relevant).

Reply:
Sorry: I should clarify that I didn't find it mentioned in the `astroid` documentation - and that's where I'd consider opening a corresponding issue/pull request.

Reply:
We're using the ``qname`` attribute generally, you can check how it's one in checker for exceptions too broad raised/caught.

Reply:
I think `ExceptionInstance` is fine here. We're the maintainers of `astroid` as well. So if we ever break the API we are also in charge of fixing it. I don't think this is an issue here.